### ISSUE TITLE
Having a meaningful exception message when the same produces call is made

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#13268]
A module had two identical `produces()` call and the exception message was
```
----- Begin Fatal Exception 11-Feb-2016 15:47:53 CET-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=TauMatchedRecoObjectProducer label='genATauMuMatchedRecoTauSelectorCJ'
Exception Message:
Duplicate Process The process name CleanJetsAnalyzer was previously used on these products.
Please modify the configuration file to use a distinct process name.
----- End Fatal Exception -------------------------------------------------
```

We should change this to make it clear that the problem is one module making two identical `produces` calls.

**Comment by Dr15Jones** [comment]
assign core

**Comment by wmtan** [comment]
@Dr15Jones Fixed with PR #13432

**Comment by Dr15Jones** [comment]
+1
